lib/Service/ZammadAPIService.php

```diff
@@ -495,25 +497,22 @@ public function request(
 				return ['error' => $this->l10n->t('Bad HTTP method')];
 			}
 			$body = $response->getBody();
-			$respCode = $response->getStatusCode();
 
-			if ($respCode >= 400) {
```

Comment:
>= 400 is ClientException and not in this code path since … many years.
So I moved it to the catch, and return the `error-code` in addition for 401 so we can react in the function call from the background job and remove it there